File: C/Daily Flash/Week2/Day3_Jan22/code/prog3.c
#include<stdio.h>

void main() {

	int num1,num2;
	printf("Enter number 1: ");
	scanf("%d",&num1);

	printf("Enter number 2: ");
	scanf("%d",&num2);
	
	int cube1 = num1*num1*num1;
	int square1 = num1*num1;

	int cube2 = num2*num2*num2;
	int square2 = num2*num2;

	//addtion of their cube
	printf("Addition of %d & %d = %d\n",cube1 , cube2 ,cube1+cube2 );

	//substraction of their square
	printf("Substraction of %d & %d = %d\n",square1 , square2 ,square1 - square2 );
}

File: PYTHON/Daily Flash/Week2/Day3_Jan22/code/prog3.py
'''
Program 3: Write a program that accepts two integers from user and prints
addition of their Cubes and subtraction of their Squares.
Input: 10 20
Output:
Addition of 1000 & 8000 is 9000
Subtraction of 100 & 400 is -300
'''

#take two num 
num1 = int(input("Enter num 1: "))
num2 = int(input("Enter num 2: "))

#calculate cube
c1 = num1*num1*num1
c2 = num2*num2*num2

#calculate square
s1 = num1*num1
s2 = num2*num2

print("Addition of " , c1 , "& ", c2 , " is " , c1+c2)
print("Substraction of " , s1 , "& ", s2 , " is " , s1-s2)
        
